doc/admin/predeployment_check.md

```diff
@@ -102,6 +95,14 @@ the relevant settings.
 
 For more information, please refer to the [librdmacm documentation](https://github.com/linux-rdma/rdma-core/blob/master/Documentation/librdmacm.md)
 
+### Subnet
+
+Since all engines need to be able to communicate, the different network
+interfaces need to be on the same subnet or routing capabilities across the
+different subnet must be configured.
```

Comment:
I don't really understand why this is moved at the bottom. Could you please elaborate?

Reply:
While going through this section with Tom, it was decided that the (previously) headed "infiniband/RoCE" section made more sense sequentially prior to the subnet section.